cmds/webboot/webboot.go

```diff
@@ -118,6 +118,7 @@ func (d *DownloadOption) exec(uiEvents <-chan ui.Event, network bool, cacheDir s
 	}
 
 	entries := supportedDistroEntries()
+	entries = append(entries, &Config{"Other Distro"})
```

Comment:
Make "Other Distro" a constant?

Reply:
Yeah that makes sense. It should be fixed in the new push.